### PR TITLE
add telegram notifier

### PR DESCRIPTION
**Pull request by mhost39**
### Description
add telegram notifier send messages to telegram group VDCs Monitoring

### Related Issues

https://github.com/threefoldtech/js-sdk/issues/3044
### Checklist

- [ ] [Pre-commit hook is installed](https://github.com/threefoldtech/js-ng#pre-commit) to do formatting checks before committing code...etc
- [ ] Tests included
- [ ] Build pass
- [ ] Documentation
- [ ] Code format and docstrings

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -33,6 +33,7 @@ requests_unixsocket = "^0.2.0"
 protobuf = "^3.12.2"
 namecom = "^0.5.0"
 crontab = "^0.22.9"
+python-telegram-bot = ""
```
Comment (waleedhammam) [inline pyproject.toml:36]: we should add version, 

```suggestion
python-telegram-bot = "^0.0.1"
```

and should be added using in order to update `poetry.lock` file aswell

```
poetry add telegram
```

**Review by waleedhammam — CHANGES_REQUESTED** [review]

**Comment by waleedhammam** [comment]
Please check the CI:

```
Because no versions of python-telegram-bot match >=13.4.1,<13.5 || >13.5,<14.0.0
   and python-telegram-bot (13.5) depends on tornado (>=5.1), python-telegram-bot (>=13.4.1,<14.0.0) requires tornado (>=5.1).
```

File: jumpscale/packages/admin/services/notifier.py [hunk 2]
```diff
@@ -36,6 +39,29 @@ def mail_notifier(self):
...
+                break
+            try:
+                telegram_msgs = j.data.serializers.json.loads(telegram_msg_json.decode())
+                self.send_telegram_msg(msg=telegram_msgs)
+
+            except Exception as e:
+                j.logger.exception(f"Failed to send Telegram message: {telegram_msg_json}", exception=e)
+
+    def send_telegram_msg(self, msg):
+        telegram_config = j.core.config.get("VDC_TELEGRAM_CHAT_CONFIG")
```
Comment (xmonader) [inline jumpscale/packages/admin/services/notifier.py:58]: TELEGRAM_TOKEN, this has nothing to do with VDC, it's a generic admin service.

File: jumpscale/packages/admin/services/notifier.py [hunk 3]
```diff
@@ -36,6 +39,29 @@ def mail_notifier(self):
             except Exception as e:
                 j.logger.exception(f"Failed to send mail: {mail_info_json}", exception=e)
 
+    def telegram_notifier(self):
+        """
+        send messages to telegram group VDCs Monitoring from redis queue.
```
Comment (waleedhammam) [inline jumpscale/packages/admin/services/notifier.py:44]: remove `VDCs` it's a generic service

**Review by waleedhammam — CHANGES_REQUESTED** [review]
Please add a quick readme / how to use. for the service

**Review by waleedhammam — APPROVED** [review]

**Pull request merged**